Header header

float32 l_R		# distance wheel right, in m
float32 l_L		# distance wheel left, in m
uint16 i_R		# motor currents	
uint16 i_L
int16 v_R		# wheel velocities caluclated by the motor driver
int16 v_L
uint16 rawR		# raw position of the wheels in ticks of the Hall Sensor A
uint16 rawL